api/src/main/java/com/alibaba/nacos/api/naming/NamingFactory.java

```diff
@@ -18,6 +18,7 @@
 import java.lang.reflect.Constructor;
 import java.util.Properties;
 
+import com.alibaba.nacos.api.config.ConfigService;
```

Comment:
Why is ConfigService in namingFactory?

Reply:
Okay, I will remove this unused import, thanks.